Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -1183,11 +1183,8 @@ void OnServerRemovePlayerMessageInternal(NetworkConnection conn, RemovePlayerMes
         {
             if (LogFilter.Debug) Debug.Log("NetworkManager.OnServerRemovePlayerMessageInternal");
 
-            if (conn.identity != null)
-            {
-                OnServerRemovePlayer(conn, conn.identity);
-                conn.identity = null;
-            }
+            OnServerRemovePlayer(conn);
```

Comment:
You should call the old `OnServerRemovePlayer` here too or set `IsError = true`  in the obsolete

Reply:
The new method does what the old one does in a new place.

Reply:
The old method is a virtual method, anyone who currently overrides it expects it to be called.  Removing the call to it is a breaking change.

Reply:
I think I'm going to make this a breaking change and be done with it.  I honestly don't think it could have been sensibly used in its current state, and if there's someone out there using it, they'll know and the change is easy.

Reply:
IsError = true in the obsolete then, so the compiler forces them to change

`Obsolete("Override OnServerRemovePlayer(NetworkConnection conn) instead", true)`